edenia/antelope-tools
# Fix/missing bp logos

### Fix missing logos in nodes page

### What does this PR do?

- Resolve #1236
- Use a custom hook to validate the bp logo
- Remove uneeded hook to translate the node type

### Steps to test

1. Run the project locally
2. Go to nodes page
3. Check that the images are displayed

### Screenshots

![imagen](https://github.com/edenia/antelope-tools/assets/66583677/a7e4142b-30ea-4258-8d26-c51098c105fa)
